python/gtsam/tests/test_PinholeCamera.py

```diff
@@ -0,0 +1,58 @@
+"""
+GTSAM Copyright 2010-2019, Georgia Tech Research Corporation,
+Atlanta, Georgia 30332-0415
+All Rights Reserved
+
+See LICENSE for the license information
+
+PinholeCamera unit tests.
+Author: Fan Jiang
+"""
+import unittest
+from math import pi
+
+import numpy as np
+
+import gtsam
+from gtsam.utils.test_case import GtsamTestCase
+
+
+class TestPinholeCamera(GtsamTestCase):
+    def test_jacobian(self):
+        cam1 = gtsam.PinholeCameraCal3Bundler()
+        cam1.deserialize(
```

Comment:
uhg? This makes it a regression test, not a unit test.
For purposes of the new functionality (and documentation on how to use) a simple example with round numbers would be preferred. It suffices even to just check one number as a regression, as this is not supposed to test the derivatives (done nicely in c++) but the passing of derivatives to python.

Reply:
done